extensions/interactions/ImageClickInput/directives/ImageClickInput.js

```diff
@@ -45,11 +45,41 @@ oppia.directive('oppiaInteractiveImageClickInput', [
             ($attrs.highlightRegionsOnHoverWithValue === 'true');
           $scope.filepath = imageAndRegions.imagePath;
           $scope.imageUrl = '';
-
-          ImagePreloaderService.getImageUrl($scope.filepath)
-            .then(function(objectUrl) {
-              $scope.imageUrl = objectUrl;
-            });
+          $scope.loadingIndicatorUrl = (
+            UrlInterpolationService.getStaticImageUrl(
+              '/activity/loadingIndicator.gif')
+          );
```

Comment:
Maybe create constant for this?

Reply:
@ishucr7 ping!

Reply:
Sorry didn't get it. Constant for loadingIndicatorUrl in feconf ?

Reply:
Done!